runtime/vm/vmthread.c

```diff
@@ -210,6 +210,16 @@ allocateVMThread(J9JavaVM * vm, omrthread_t osThread, UDATA privateFlags, void *
 	newThread->mgmtWaitedStart = JNI_FALSE;
 #endif
 
+#ifdef OMR_GC_CONCURRENT_SCAVENGER
+	/* Initialize fields used by Concurrent Scavenger */
+	newThread->evacuateTop = NULL;
+	newThread->evacuateBase = (void*)~(UDATA)newThread->evacuateTop;
+#ifdef J9VM_GC_COMPRESSED_POINTERS
+	newThread->evacuateTopCompressed = 0;
+	newThread->evacuateBaseCompressed = ~newThread->evacuateTopCompressed;
+#endif
```

Comment:
nitpick - add the closing `/* J9VM_GC_COMPRESSED_POINTERS */` to endif's

Reply:
Updated.